### ISSUE TITLE
Inconsistency of Email Notification Level and Email Address Required

### ISSUE THREAD

**Issue by cougar2009** [awslabs/aws-limit-monitor#27]
Email Notification Level
List of alert levels to send email alerts in response to. Leave blank if you do not wish to receive email notifications. Must be double-quoted and comma separated.

Email Address
(Required) The email address to subscribe for alert messages.

It doesn't make sense to me to have an option in email notification level to leave it blank if we don't want notifications and then to also require that an email address be present.   

also if I do try to leave it blank for Email Notification Level I get the following error:

Template format error: Unresolved resource dependencies [SNSTopic] in the Resources block of the template

so basically it an Email Notification Level appears to be required.

**Comment by beomseoklee** [comment]
Hi @cougar2009, and we apologize for the inconvenience. We are currently researching the issue at our end and  working towards the resolution. Please continue to monitor the repository for updates.

**Comment by gsingh04** [comment]
Thanks for your feedback. Essentially, when you don't want email notifications, you should leave **Email Notification Level** and **Email Address** blank. However, there is bug in the template which results in an undesired dependency https://github.com/awslabs/aws-limit-monitor/blob/1dcd25b9ffbb2e8b2f230bd6650dc236fb779b5e/deployment/limit-monitor.template#L436

We need to add condition for this resource as well. 
`Condition: SNSTrue`

We will make this change with next push.

**Comment by georgebearden** [comment]
This issue should have been addressed in the latest commit. Please let us know if you need additional assistance.
